Formula/zstd.rb

```diff
@@ -8,6 +8,12 @@ class Zstd < Formula
   license "BSD-3-Clause"
   head "https://github.com/facebook/zstd.git", branch: "dev"
 
+  livecheck do
+    url "https://github.com/facebook/zstd/"
+    regex(%r{href=["']?[^"' >]*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+    strategy :page_match
```

Comment:
```suggestion
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```
Wouldn't this work if what we want is just to avoid the `1.5.2-kernel` tag?